XCUITests/FxScreenGraph.swift

```diff
@@ -606,6 +606,7 @@ extension Navigator where T == FxUserState {
     }
 
     func openURL(_ urlString: String, waitForLoading: Bool = true) {
+        UIPasteboard.general.string = "http://example.com"
```

Comment:
What? Why don't you use `urlString` instead of `example.com`?

Reply:
Sorry! just changed it, leave that to try and forgot to change